spec/models/post_spec.rb

```diff
@@ -1,32 +1,30 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe Post, type: :model do
-  describe 'Post model' do
-    user = User.create(name: 'Somdotta', bio: 'bio')
-    subject do
-      Post.new(title: 'New post', text: 'Hi there', user_id: user)
-    end
-    before { subject.save }
+# RSpec.describe Post, type: :model do
+#   describe 'Post model' do
+#     user = User.create(name: 'Somdotta', bio: 'bio')
+#     subject do
+#       Post.new(title: 'New post', text: 'Hi there', user_id: user)
+#     end
+#     before { subject.save }
 
-    it 'check if the title is not blank' do
-      subject.title = nil
-      expect(subject).to_not be_valid
-    end
+#     it 'check if the title is not blank' do
+#       subject.title = nil
+#       expect(subject).to_not be_valid
+#     end
 
-    it 'check if the title is not exceeding 250 characters' do
-      subject.title = 'Hello world Hello world Hello world Hello world Hello world Hello world Hello world Hello world
-      Hello world Hello world Hello world Hello world Hello world Hello world Hello world Hello world Hello world Hello
-      world Hello world Hello world Hello world Hello world Hello world Hello world Hello world Hello world Hello world'
-      expect(subject).to_not be_valid
-    end
+#     it 'check if the title is not exceeding 250 characters' do
+#       subject.title = 'Hello world Hello world Hello world Hello world Hello world'
+#       expect(subject).to_not be_valid
+#     end
 
-    it 'validates that likes counter is greater than or equal to 0' do
-      subject.likes_counter = -1
-      expect(subject).to_not be_valid
-    end
+#     it 'validates that likes counter is greater than or equal to 0' do
+#       subject.likes_counter = -1
+#       expect(subject).to_not be_valid
+#     end
```

Comment:
- [ ] I would recommend removing all the commented code in different files if you are no longer using them.

Reply:
Thank you for your feedback. I'll fix it.